www/src/css/components/_card-grid.scss

```
@@ -0,0 +1,105 @@
+.card-grid {
+  display: grid;
+  grid-column-gap: 15px;
+  grid-row-gap: 15px;
+  grid-auto-flow: dense;
+}
+
+.card-grid-3 {
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+.card-grid-4 {
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+}
+.discord-banner {
+  grid-column: 1 / -1;
+  border: 1px solid #2e2077;
+  background-color: #545eec;
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  color: white;
+  font-weight: 500;
+  padding: 1.25rem;
+  margin: 1.5rem 0 1rem;
+  background: #545eec;
+  box-shadow: 6px 6px 20px #4750c966, -6px -6px 20px #616cff66;
+}
+.discord-banner > * {
+  display: block;
+}
+
+@media (max-width: 700px) {
+  .discord-banner {
+    flex-direction: column;
+  }
+  .discord-banner > div {
+    margin-top: 1rem;
+  }
+}
+
+.card {
+  display: flex;
+  grid-column: span 1;
+  overflow: hidden;
+  font-family: Open Sans, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI,
```

```suggestion
  font-family: "Open Sans", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI",
```
2-word font family names should be quoted. I don’t think browsers actually have a problem with this, and I think it works either way, but just to be safe.